perception/lidar_centerpoint/src/node.cpp

```diff
@@ -14,6 +14,9 @@
 
 #include "lidar_centerpoint/node.hpp"
 
+#include "perception_utils/perception_utils.hpp"
```

Comment:
```
#include "perception_utils/perception_utils.hpp"
#include "tier4_autoware_utils/tier4_autoware_utils.hpp"
```
seem to be unnecessary

Reply:
removed at [21cb9f5](https://github.com/autowarefoundation/autoware.universe/pull/1935/commits/21cb9f5be1bcd1db097501d7c9f3adb56abb51c4).